templates/default/stunnel.conf.erb

```diff
@@ -63,6 +63,11 @@ debug = <%= node['stunnel']['debug'] %>
 output = <%= node['stunnel']['output'] %>
 <% end -%>
 
+; Enable/disable FIPS mode
+<% unless node['stunnel']['fips'].nil? %>
```

Comment:
Hi Aaron, I appreciate the feedback. The reason I went with unless .. .nil? is because there are some circumstances where you do not want the "fips" option to be added to the stunnel config. For example, if FIPS support isn't compiled into OpenSSL or stunnel (which appears to be the case on Ubuntu 14.04) the daemon will not start.

So the ['stunnel']['fips'] attribute has 3 values:
- nil is don't include the "fips" option in the stunnel config at all. This is the default behavior and should not break backwards compatibility for anyone using the cookbook.
- true/false set it to yes/no

Does that make sense? Is there some better way to handle that? (I will, of course, make the other changes you've requested.)